app/models/matchup.rb

```diff
@@ -16,14 +17,15 @@
 #  updated_at       :datetime         not null
 #
 class Matchup < ApplicationRecord
-  validates :year, :round, :conference, :number, :favorite_tricode, :underdog_tricode,
+  validates :sport, :year, :round, :conference, :number, :favorite_tricode, :underdog_tricode,
             :favorite_wins, :underdog_wins, :starts_at, presence: true
+  validates :sport, inclusion: {in: %w[nba mlb]}
   validates :year, numericality: {equal_to: 2021}
   validates :round, numericality: {
     greater_than_or_equal_to: 1, less_than_or_equal_to: 4
   }
-  validates :conference, inclusion: {in: %w[east west], if: -> { round < 4 }},
-                         exclusion: {in: %w[east west], if: -> { round == 4 }}
+  validates :conference, inclusion: {in: %w[east west AL NL], if: -> { round < 4 }},
+                       exclusion: {in: %w[east west AL NL], if: -> { round == 4 }}
   validates :number, uniqueness: {scope: %i[year round conference]},
```

Comment:
Need this or else it will be overly restrictive.
```suggestion
  validates :number, uniqueness: {scope: %i[sport year round conference]},
```